src/pages/GradingSystem.tsx

```diff
@@ -0,0 +1,254 @@
+/* eslint-disable */
+import React, { useEffect, useState } from 'react';
+import Sidebar from '../components/Sidebar';
+import { useTranslation } from 'react-i18next';
+import grade from '../dummyData/gradingSystem.json';
+import Button from '../components/Buttons';
+
+const GradingSystem = () => {
+  const [addGradingSystemModel, setAddGradingSystemModel] = useState(false);
+  const { t } = useTranslation();
+  const [title, setTitle] = useState('');
+
+  let fileteredData;
+  const [formData, setFormData] = useState({
+    label: '',
+  });
+
+  if (formData.label === 'gpa') {
+    fileteredData = grade.filter((item: any) => item.label === formData.label);
+  } else if (formData.label === 'letter') {
+    fileteredData = grade.filter((item: any) => item.label === formData.label);
+  } else if (formData.label === 'atlp') {
+    fileteredData = grade.filter((item: any) => item.label === formData.label);
+  }
+
+  useEffect(() => {
+    if (formData.label === 'gpa') {
+      setTitle('Grade Point Average');
+    }
+    if (formData.label === 'letter') {
+      setTitle('Letter Grading');
+    }
+    if (formData.label === 'atlp') {
+      setTitle('ATLP Grading System');
+    }
+    if (formData.label === 'atlp') {
+      setTitle('ATLP Grading System');
+    }
+    if (formData.label === 'custom') {
+      setTitle('Custom Name');
+      setAddGradingSystemModel(true);
+    }
```

Comment:
How does one access this interface to add a new grading system?

Reply:
I'm not sure why you marked this as resolved yet I see no action taken or comment on this thread.

Reply:
Sorry @ceelogre 
I thought I left comments on this thread, I had a headache that's why I didn't check twice

How does one access this interface to add a new grading system?
There is a model which allows users to customize their grading system, Other than that
I thought it would be addressed in the backend side, when we will be doing integration

<img width="1440" alt="Screen Shot 2022-08-12 at 05 23 49" src="https://user-images.githubusercontent.com/87600918/184280568-2e6fff89-46f8-40cb-9714-3a6b1e473c09.png">